Partition (WBP)/Functions/VI0049D.VTI0049B.fun.yml

```diff
@@ -0,0 +1,135 @@
+﻿Function:
+  Enabled For RDMLX:  Yes
+
+  Source: |
+    <-
+    
+    Function Options(*DIRECT) Rcv_List(#wLKData #wLRet)
+    
+    * *********************************************************************************************
+    * Test Case Description
+    * Issue#49 Continued : PerformanceMetrics data Table generation.
+    * VTI0049B will be calling Function VI0049D which the pipeline will execute after VTI0049.
+    * The test setup will reformat 'PerformanceMetricsLog.txt' into 'PerformanceMetricsTable.xls'.
+    * This test case will be write the comma separated contents of a text file into an xls file through a list using TRANSFOMR_FILE and TRANSFORM_LIST.
+    * *********************************************************************************************
+    
+    * Unique Key Data.
+    Def_List Name(#wLKData) Fields(#VTF_KEY1 #VTF_KEY2 #VTF_KEY3) Type(*WORKING) Entrys(1)
+    
+    * Return messages.
+    Def_List Name(#wLRet) Fields(#BIFMSGC #BIFMSG1 #BIFMSG2) Counter(#BIFMSGCNT) Type(*WORKING) Entrys(*MAX)
+    
+    Define Field(#LF_i) Reffld(#Std_Count)
+    Define Field(#LF_Count) Reffld(#Std_Count)
+    Define Field(#LF_rc) Reffld(#BIFA002A) Default(OK)
+    Define Field(#LF_logtablefile_name) Reffld(#BIFA256A)
+    
+    * Start testing.
+    Clr_List Named(#wLRet)
+    #Com_Owner.mthAdd_Msg( OK *Function ('Start - ' + *Function_Desc) )
+    #Com_Owner.mthGetExecutionType()
+    * Export the contents of log file into an xls file only if the log file exists
```

Comment:
As I said in the meeting you need to condition it like this!
```
   If (*OSBITNESS *EQ 32)
        If ((#LF_Execution_Type *EQ W) *And (#LF_Local_Platform *EQ W) *And (#LF_DB_Type *EQ MSSQLS))
```

Reply:
Thank you @robe070 . I will incorporate the above comments.